Electron-Basic-Screen-Sharing/README.md

```diff
@@ -0,0 +1,194 @@
+OpenTok.js Electron Basic Screen-Sharing Sample (Electron 17)
+=======================
+
+This sample application shows how to connect to an OpenTok session, publish a screen-sharing stream, and subscribe to a stream in a basic Electron application.
+
+## Quick started
+1. Get values for your OpenTok API key, session ID, and token. You can obtain these values from your TokBox account. Make sure that the token isn't expired.
+For testing, you can use a session ID and token generated at your TokBox account page. However, the final application should obtain these values using the [OpenTok server SDKs](https://tokbox.com/developer/sdks/server/). For more information, see the OpenTok developer guides on [session creation](https://tokbox.com/developer/guides/create-session/) and [token creation](https://tokbox.com/developer/guides/create-token/).
+
+2. `$ yarn install`
+
+3. `$ yarn start`
+
+*Important:* Read the following sections of the of the README file to get started
+
+## Installing Dependencies with `yarn`
+
+`$ yarn install`
+
+## Getting an OpenTok session ID, token, and API key
```

Comment:
There's a lot of stuff in here, do you think we need all this? 
I'm looking at other samples and tbh we're not very consistent with the level of info in the readme's so I'm not sure what's best. 
On the plus-side, someone who has no idea about opentok and wants to build this has all the info in one places. 
The down-side being that we have this stuff repeated all over the place, having to maintain it several times over when we could just refer someone to the Basic Video Chat sample, and focus on screen-sharing specific stuff here.

Reply:
Since this was similar to the Electron basic video chat sample, I kept most of that README in this one. I agree; it's a little too much.

Let's go with maintainability. I'll tailor this README to focus on screen-sharing and add a reference in the beginning for the Basic Video Chat app too.

Please let me know what you think of the next commit!